agent/core/test/Page.test.ts

```diff
@@ -201,6 +207,7 @@ describe('Pages', () => {
          <div style="height: 200px; width: 100px; background:red">&nbsp;</div>`,
       );
       const screenshot = await page.screenshot({ format: 'png' });
+      writeFileSync(`${screenshotDir}/1-viewport.png`, screenshot);
```

Comment:
Do you think there's any reason to keep these assets? I understood the purpose when we were testing, but I'm not sure they add much value as test outputs. Open to something I'm missing though!

Reply:
Not really, could help if they break it again, mainly to see if the content is blank or as expected. But I agree this clutters the tests, and if needed can easily be added again. Will remove them.